If grouping_columns includes int typed column, the plugin gets NoMethodError "undefined method `split' for 1:Fixnum"

The version of the plugin is v0.2.0.

```
Caused by: org.jruby.exceptions.RaiseException: (NoMethodError) undefined method `split' for 1:Fixnum
	at RUBY.block in add_subscriber(gems/embulk-output-mailchimp-0.2.0/lib/embulk/output/mailchimp.rb:121)
	at org.jruby.RubyArray.each(org/jruby/org/jruby/RubyArray.java:1560)
	at RUBY.add_subscriber(gems/embulk-output-mailchimp-0.2.0/lib/embulk/output/mailchimp.rb:118)
	at RUBY.block in add(gems/embulk-output-mailchimp-0.2.0/lib/embulk/output/mailchimp.rb:83)
	at RUBY.each(uri:classloader:/embulk/page.rb:22)
	at RUBY.add(gems/embulk-output-mailchimp-0.2.0/lib/embulk/output/mailchimp.rb:80)
	at RUBY.add(uri:classloader:/embulk/output_plugin.rb:114)
```

--- a/lib/embulk/output/mailchimp.rb
+++ b/lib/embulk/output/mailchimp.rb
@@ -127,11 +127,12 @@ module Embulk
           merge_vars[:groupings] = []
 
           @grouping_columns.each do |group|
-            next if row[group].nil? || row[group] == ''
+            groups = row[group].to_s.split(",")
+            next if groups.empty?
 
             merge_vars[:groupings] << {
               'name'   => group,
-              'groups' => row[group].split(','),
+              'groups' => groups,
             }
           end
         end

--- a/test/embulk/output/test_mailchimp.rb
+++ b/test/embulk/output/test_mailchimp.rb
@@ -125,6 +125,28 @@ module Embulk
               ]
             )
           end
+
+          def test_grouping_columns_not_string
+            groups_1 = 42
+            groups_2 = 1.3
+
+            mailchimp = create_mailchimp(
+              lname_column: 'lname',
+              fname_column: 'fname',
+              grouping_columns: ['list_group1', 'list_group2']
+            )
+
+            mailchimp.add([['i@example.com', 'lname', 'fname', groups_1, groups_2]])
+            subscriber = mailchimp.instance_variable_get(:@subscribers)[0]
+
+            assert_equal(
+              subscriber[:merge_vars][:groupings],
+              [
+                {'name' => 'list_group1', 'groups' => [groups_1.to_s] },
+                {'name' => 'list_group2', 'groups' => [groups_2.to_s] },
+              ]
+            )
+          end
         end
 
         # NOTE skip until const mocking

